services/QuillLMS/app/services/demo/report_demo_creator.rb

```diff
@@ -128,9 +131,24 @@ def self.create_classroom_units(classroom, unit)
         assign_on_join: true)
   end
 
+  def self.create_replayed_activity_session(student)
+    temp = ActivitySession.unscoped.where({activity_id: REPLAYED_ACTIVITY_ID, user_id: REPLAYED_SAMPLE_USER_ID, is_final_score: true}).first
+    cu = ClassroomUnit.where("#{student.id} = ANY (assigned_student_ids)").to_a.first
```

Comment:
FYI, you can use AREL syntax here, like 

`.where(studentId: assigned_student_ids)`